packages/pynumaflow/examples/sink/all_sinks/entry.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I don't think we need this file. Update `Dockerfile` to directly invoke the Python command. Let's try to make the examples simpler.